### DISCUSSION TITLE
scaleToWidth() function not working on mobile device.

### DISCUSSION THREAD

**Discussion by vipin-vidyut** [fabricjs/fabric.js#8464] [General]
scaleToWidth() function not working on mobile device.

`canvas.discardActiveObject();
var objs = canvas.getObjects();
sel = new fabric.ActiveSelection(canvas.getObjects(), {
        canvas: canvas,
      });
      sel.scaleToWidth(Number(250),true);
      canvas.requestRenderAll();`
when this executes all object on canvas disappear.

**Comment by ShaMan123** [comment]
Please add reproduction from now on.
You didn't attach the active selection to canvas.
